src/components/sources-list/sources-list.tsx

```diff
@@ -70,7 +74,7 @@ export const SourcesList = withText({
       } else if (flavor.height >= HeightResolution.HD) {
         flavorDescription += ` ${RESOLUTION_HD}`;
       }
-      if (flavor.isOriginal) {
+      if (flavor.isOriginal || flavor.flavorParamsId === 0) {
```

Comment:
flavorParamsId 0 has a special meaning ? it will exist in every environment ?

Reply:
flavorParamsId 0 is the source flavor. so if the flavor is source or it has isOriginal=true, the description text should include "Source" in addition to the resolution.